src/transformers/tokenization_utils_base.py

```diff
@@ -1521,6 +1521,8 @@ def __init__(self, **kwargs):
 
         self.model_input_names = kwargs.pop("model_input_names", self.model_input_names)
 
+        # By default, cleaning tokenization spaces for both fast and slow tokenizers
+        self.clean_up_tokenization_spaces = kwargs.pop("clean_up_tokenization_spaces", True)
```

Comment:
Will that get sent from the config ? 

I think we should add a tiny tokenizer with the config on, and confirm the thing loads a decodes properly.

Wdyt.

Then `bloom` models, `llama`  (and maybe some `gpt2`) can be updated.

Reply:
Sure! Updating in the modeling would mean a breaking changes, but if it is what we want I won't be against it

Reply:
Wdymean the modeling ? 

I meant make sure that we can actually configure a tokenizer such that this is seemless. `tokenizer_config.json` works for me (just need to make sure it actually flows)

Reply:
Adding a test with tiny config to make sure things workout properly!